README.md

```diff
@@ -1,4 +1,11 @@
-# A11y Workshop
+# Accessible Development with JavaScript and HTML5
+
+Learn how to design, test, and build accessible UI components with HTML5, Sass,
+and jQuery, demonstrated by designers and developers at [thoughtbot](http://thoughtbot.com).
+
+In the process, learn about types of disabilities, motivations for accessibility,
+and standards and requirements, along with testing tools and accessible UI
```

Comment:
"standards and requirements" strikes me as more boring and paperwork-y, while "motivations for accessibility" seems like something I may want to learn about: "what am I missing?". What do you think of mentioning in the reverse order, so we end the sentence in a higher note?